spec/factories/approvals.rb

```diff
@@ -1,10 +1,10 @@
 FactoryGirl.define do
```

Comment:
should we move this factory since we now have the exact same thing at `steps.rb` ?